site/src/components/tabs.md

```diff
@@ -0,0 +1,144 @@
+---
+title: Tabs
+description: Tabs component in the Atlas Design System
+template: standard
+classType: Component
+classPrefixes:
+  - tabs
+---
+
+## Tabs
+
+Tabs component consists of navigating to the previous or next tab and well and clicking on a tab. The result will change the content in the `tab-item-container` to display the current selected item. To do this the `[data-tab-control="1"]` and `[data-tab-item="1"]` must have matching numbers as these act as the index for navigation.
```

Comment:
"Tabs component consists of navigating to the previous or next tab and well and clicking on a tab" 
- as well as clicking on a tab?

Reply:
updated